docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+is_result_mounted=`mount | grep "$RESULT_DIR"`
+if [ -z "is_result_mounted" ]
+then
+    echo "Output directory for the result is not mounted, so we can't save the result"
```

Comment:
What about more detailed message with description how to run Docker so the output directory is mounted?